crates/rpc/src/error.rs

```diff
@@ -1,27 +1,34 @@
 use beerus_core::CoreError;
 use eyre::Report;
-use jsonrpsee::core::Error;
-use jsonrpsee::types::error::ErrorCode;
+use jsonrpsee::core::Error as JsonRpcError;
 use jsonrpsee::types::ErrorObjectOwned;
 use starknet::core::types::StarknetError;
 use starknet::providers::ProviderError;
 use starknet::providers::ProviderError::StarknetError as StarknetProviderError;
 use thiserror::Error;
 
 #[derive(Error, Debug)]
-pub enum BeerusRpcError {
+pub enum RunError {
+    #[error("wrong RPC spec version: expected {1} but got {0}")]
```

Comment:
We should clarify that the `got` version is the remote one.
Maybe we could also mention the relevant config key, as well as the "Version support" column from [this table](https://docs.starknet.io/documentation/tools/api-services/)?